DOCUMENTATION.md

```diff
@@ -1325,18 +1318,29 @@ clientConfig.networkConfig.connectionTimeout = 6000;
 
 Its default value is `5000` milliseconds.
 
-## 5.5. Setting Connection Attempt Limit
+## 5.5. Enabling Client TLS/SSL
+
+You can use TLS/SSL to secure the connection between the clients and members. If you want to enable TLS/SSL
+for the client-cluster connection, you should set an SSL configuration. Please see [TLS/SSL section](#81-tlsssl).
+
+As explained in the [TLS/SSL section](#81-tlsssl), Hazelcast members have key stores used to identify themselves (to other members) and Hazelcast Node.js clients have certificate authorities used to define which members they can trust. Hazelcast has the mutual authentication feature which allows the Node.js clients also to have their private keys and public certificates, and members to have their certificate authorities so that the members can know which clients they can trust. See the [Mutual Authentication section](#813-mutual-authentication).
 
-While the client is trying to connect initially to one of the members in the `ClientNetworkConfig.addresses`, that member might not be available at that moment. Instead of giving up, throwing an error and stopping the client, the client will retry as many as `ClientNetworkConfig.connectionAttemptLimit` times. This is also the case when the previously established connection between the client and that member goes down.
+## 5.6. Enabling Hazelcast Cloud Discovery
+
+The purpose of Hazelcast Cloud Discovery is to provide the clients to use IP addresses provided by `hazelcast orchestrator`. To enable Hazelcast Cloud Discovery, specify a token for the `discoveryToken` field and set the `enabled` field to `true`.
```

Comment:
nit: leave https://cloud.hazelcast.com link somewhere here, so users can quickly navigate to Hazelcast Cloud website.